src/quickstart/provider/minikube/dns.md

```diff
@@ -9,53 +9,53 @@ Kubernetes overlay network.
 First, determine the ip address allocated to your worker node.
 
 ```
-$ kubectl get nodes
-NAME                STATUS    AGE
-kubernetes-node-1   Ready     4h
-$ kubectl describe node kubernetes-node-1 | egrep Addresses
-Addresses:      10.245.1.3,10.245.1.3
+$ minikube ip
+192.168.99.100
 ```
 
-Here, our node address is `10.254.1.3`.
+Here, our node address is `192.168.99.100`.
 
 Next, determine the ip address of the Deis Workflow router component:
 
 ```
 $ kubectl --namespace=deis describe service deis-router | egrep IP
-IP:                     10.247.114.249
+IP:                     10.0.0.87
 ```
 
-The service address for the router is `10.247.114.249`.
+The service address for the router is `10.0.0.87`.
 
 Last, we need to inform your machine how to reach the service address.
 
 Add routes on Linux with:
+
 ```
-$ sudo route add 10.247.114.249 gw 10.245.1.3
+$ sudo route add 10.0.0.87 gw 192.168.99.100
```

Comment:
Is adding routes really necessary? We suggest installing Workflow with `helm install --set router.host_port.enabled=true`, which should make things accessible at `$(minikube ip)`.

I just do: `$ deis register http://deis.$(minikube ip).nip.io`